tests/regression/aruba/aruba_test.py

```diff
@@ -875,6 +895,39 @@ def testNegateAny(self):
         self.assertEqual(textwrap.dedent(expected_result), str(aru))
         print(aru)
 
+    def testPlatformTerm(self):
+        self.naming.GetNetAddr.return_value = [
+            nacaddr.IP('100.0.0.0/8'),
+            nacaddr.IP('10.0.0.1/32'),
+        ]
+        self.naming.GetServiceByProto.return_value = ['69']
+        aru = aruba.Aruba(
+            policy.ParsePolicy(
+                GOOD_HEADER_V4 + GOOD_TERMS_COMBINED_SINGLE_CASE + PLATFORM_TERM, self.naming
+            ),
+            EXP_INFO,
+        )
+        output = str(aru)
+        self.assertNotIn('platform-term', output, output)
+        print(output)
```

Comment:
Not a blocker: it looks like the `@capture.stdout` decorator is missing. Including it for this test and testPlatformExclude would cause .ref files to be created for these tests.